.cz-config.js

```diff
@@ -0,0 +1,66 @@
+module.exports = {
+  // add additional standard scopes here
+  scopes: [{name: 'accounts'}, {name: 'admin'}],
+  // use this to permanently skip any questions by listing the message key as a string
+  skipQuestions: [],
+
+  /* DEFAULT CONFIG */
+  messages: {
+    type: 'What type of changes are you committing:',
+    scope: '\nEnlighten us with the scope (optional):',
+    customScope: 'Add the scope of your liking:',
+    subject: 'Write a short and simple description of the change:\n',
+    body: 'Provide a LONGER description of the change (optional). Use "|" to break new line:\n',
+    breaking: 'List any BREAKING CHANGES (optional):\n',
+    footer: 'List any ISSUES CLOSED by this change (optional). E.g.: #31, #34:\n',
+    confirmCommit: 'Are you sure you the above looks right?',
+  },
+  types: [
+    {
+      value: 'fix',
+      name: '🐛   fix:        Changes that fix a bug',
+      emoji: '🐛',
+    },
+    {
+      value: 'feat',
+      name: ' 🚀  feat:       Changes that introduce a new feature',
+      emoji: '🚀',
+    },
+    {
+      value: 'refactor',
+      name: '🤷‍♂️   refactor:   Changes that fix a bug nor adds a feature',
```

Comment:
this emoji doesnt show on my machine

Reply:
preference?

https://github.com/ngryman/cz-emoji/blob/master/lib/types.json

Reply:
magnifying glass